sdk/storage/Azure.Storage.Blobs.Batch/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Storage Blobs Batch client library for .NET
 
-> Server Version: 2019-02-02
+> Server Version: 2019-07-07
```

Comment:
Should this say `> Server Versions: 2019-07-07 and 2019-02-02` (ditto on the others)?

Reply:
Fixed everywhere.